test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -63,3 +60,20 @@ export function setup() {
 		});
 	});
 }
+
+async function openAndRunNotebook(app: Application, filename: string): Promise<void> {
+	await app.workbench.sqlNotebook.openFile(filename);
+	await app.workbench.sqlNotebook.waitForKernel('Python 3');
```

Comment:
This should be made more generic (ex. waiting for 'Loading kernel' to be gone). But since all our notebooks now are Python, we can update later when different kernel tests are added.

Reply:
Good idea! I'll do that in a follow up PR